package.json

```diff
@@ -170,6 +169,7 @@
 		"mini-css-extract-plugin": "^0.10.0",
 		"null-loader": "^4.0.0",
 		"ora": "^4.0.2",
+		"postcss": "^7.0.35",
```

Comment:
Postcss v8 was being included by sanitize-html, and that was satisfying the peer dependency requirement that stylelint has for postcss >=7. Without sanitize-html, we need to include postcss ourselves.